common/src/main/java/bisq/common/setup/CommonSetup.java

```diff
@@ -38,6 +38,9 @@ public static void setup(UncaughtExceptionHandler uncaughtExceptionHandler) {
 
         if (Utilities.isLinux())
             System.setProperty("prism.lcdtext", "false");
+
+        // #3990 allow IPv6 custom Bitcoin node in settings by hostname
+        System.setProperty("java.net.preferIPv6Addresses", "true");
```

Comment:
Is there a drawback when we set this as default for all nodes?

Reply:
Compared to doing it based on a user preference.

Reply:
I have not seen any drawback from setting this, even if IPv6 is not available.  Here's a [document](https://docs.oracle.com/javase/1.5.0/docs/guide/net/ipv6_guide/index.html) on the subject, which mentions setting the property.  And [another one](https://stackoverflow.com/questions/20262587/where-to-set-java-net-preferipv6addresses-true#20262705) which mentions that the setting has to be made very early in app startup.

If you think its too risky we can revert to choosing between IPv4 and IPv6 addresses in BtcNodeConverter where it creates a PeerAddress from hostname.